### DISCUSSION TITLE
python-torchaudio-rocm AUR package

### DISCUSSION THREAD

**Discussion by fzadikian** [rocm-arch/rocm-arch#951] [Support] [answered]
Hello, I am developing a [python-torchaudio-rocm AUR package](https://github.com/FranGamer1892/python-torchaudio-rocm/blob/master/PKGBUILD). The ROCm stack and pytorch are working (I successfully built pytorch-torchvision-rocm, for example). However, I get many error messages like this one when trying to build torchaudio:
https://paste.debian.net/1271920
Does anyone know what could be the issue? Thanks.

**Comment by fzadikian** [comment, marked as answer]
Solved. The LoadHIP.cmake file from the torchaudio repository was too old. I just had to replace it with the new one from pytorch (minor adaptations).

PS: The package is already in the AUR, just in case anyone needs it.

**Comment by acxz** [comment]
@FranGamer1892 if you want we could host the PKGBUILD here on the rocm-arch repo like we do for https://github.com/rocm-arch/python-torchvision-rocm and https://github.com/rocm-arch/python-jax-rocm

We'll give you repo access and you can push to the repo and sync changes to the AUR

**Reply by fzadikian** [reply]
Alright sure! I'd also appreciate if other people could test the PKGBUILD on their machines.

**Reply by acxz** [reply]
Created the repo and sent you an invite. You'll have to do one more thing. Add and pin a comment like this one at the AUR page: https://aur.archlinux.org/packages/python-torchvision-rocm#comment-861606 (but with the URL link for this project)

This is so users can submit issues, PRs, and support questions on the rocm-arch Discussions page to get more visibility.

Just keep the github repo and AUR synced (i.e. only push to the AUR once you have pushed to the github) and things should run smoothly! (the first sync will be in the opposite direction as you sync the github with the AUR remote)

thanks for maintaining another package in the rocm-arch community!

**Reply by fzadikian** [reply]
No problem! I already did all of the above, plus archiving my personal repo and linking to the new one. One thing that happened is that I synced the new github repo with the AUR remote, and the AUR remote didn't have all the commits previous to the first push to the AUR. However, I don't think it's that important, because the people that worked on the cuda torchaudio package are properly credited on the PKGBUILD anyway.
